cardano-api/src/Cardano/Api/ProtocolParameters.hs

```diff
@@ -936,33 +935,33 @@ toShelleyPParamsUpdate
     , protocolUpdateTreasuryCut
     } =
     ShelleyPParams {
-      _minfeeA     = noInlineMaybeToStrictMaybe protocolUpdateTxFeePerByte
-    , _minfeeB     = noInlineMaybeToStrictMaybe protocolUpdateTxFeeFixed
-    , _maxBBSize   = noInlineMaybeToStrictMaybe protocolUpdateMaxBlockBodySize
-    , _maxTxSize   = noInlineMaybeToStrictMaybe protocolUpdateMaxTxSize
-    , _maxBHSize   = noInlineMaybeToStrictMaybe protocolUpdateMaxBlockHeaderSize
+      _minfeeA     = maybeToStrictMaybe protocolUpdateTxFeePerByte
```

Comment:
Was this rename `noInlineMaybeToStrictMaybe`->`maybeToStrictMaybe` targeted for this PR?

Does it also mean that you were able to confirm that the bug mentioned in this https://github.com/input-output-hk/cardano-node/pull/4903#issuecomment-1454238434 was fixed?

Reply:
Sorry, accidentally included this change in the PR.  The change is removed.